src/main/java/dk/kb/discover/webservice/KBAuthorization.java

```diff
@@ -0,0 +1,54 @@
+/*
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+package dk.kb.discover.webservice;
+
+import io.swagger.annotations.AuthorizationScope;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Mark a given method as requiring OAuth2 authorization with the given roles.
+ * <p>
+ * The role "public" always means "access for all" and is not checked against user roles.
+ * The role "any" means that any user role is accepted when verifying the access token:
+ * It is up to the implementation to determine access based on user roles or other
+ * information from the access token.
+ * <p>
+ * Copied and slightly adjusted from the standard swagger Authorization.
+ */
+@Target({ElementType.METHOD})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface KBAuthorization {
+    /**
+     * The role "public" always means "access for all" and is not checked against user roles.
+     * Access is granted even if no access token is provided or the access token fails validation.
+     */
+    String PUBLIC = "public";
+    /**
+     * The role "any" means that any user role is accepted when verifying the access token:
+     */
+    String ANY = "any";
```

Comment:
Probably, but no need to define roles we do not know exists. 
The two roles here are also a bit confusing since ANY actually means they need a token. But this what Toke has defined in the documentation. In 2025/2026 we probably not a lot more about roles and maybe we have refactored Oath implementation anyway.